### ISSUE TITLE
Syntax mistake in enum documentation

### ISSUE THREAD

**Issue by shajanjp** [microsoft/TypeScript-Website#283]
Syntax for enum data type is wrong on page [Basic Types](https://www.typescriptlang.org/v2/docs/handbook/basic-types.html)
Open brace is missing.
![Screenshot from 2020-02-25 11-34-37](https://user-images.githubusercontent.com/2943609/75219477-e24a1800-57c2-11ea-9c1e-053794fe250f.png)

**Comment by lgordey** [comment]
I guess it's [twoslasher](https://github.com/microsoft/TypeScript-Website/tree/v2/packages/ts-twoslasher) bug or not? 🤔 

Source code looks correct
```ts twoslash
enum Color {
  Red,
  Green,
  Blue
}
let c: Color = Color.Green;
```

**Comment by orta** [comment]
Yeah, I think this is a twoslash bug - I need to get https://github.com/microsoft/TypeScript-Website/issues/202 in which should fix a bunch of weird errors like this

**Comment by orta** [comment]
Confirmed this is fixed by #405 - http://www.staging-typescript.org/docs/handbook/basic-types.html#enum
